### DISCUSSION TITLE
GPU version

### DISCUSSION THREAD

**Discussion by ypanhtu** [Tomoki-YAMASHITA/CrySPY#70] [Q&A]
Dear Dr. Yamasita,

Does the CrySPY now support the GPU calculation? 

Thanks a lot.

Roc

HTU

**Comment by Tomoki-YAMASHITA** [comment]
CrySPY does not perform time-consuming calculations.
External programs such as VASP and QE are used for structural optimization.
So GPU support depends on the external program you use.

**Reply by ypanhtu** [reply]
Yes,  I have called the external GPU program. Thank you Tomoki.
